frontend/src/app/main/ui/workspace/tokens/modals/themes.cljs

```diff
@@ -249,16 +248,20 @@
     (ctob/activate-theme tlib (:group theme) (:name theme))))
 
 (mf/defc edit-create-theme*
-  [{:keys [set-state theme on-save is-editing]}]
+  [{:keys [state change-view theme on-save is-editing]}]
   (let [ordered-token-sets (mf/deref refs/workspace-ordered-token-sets)
         token-sets (mf/deref refs/workspace-token-sets-tree)
 
         current-theme* (mf/use-state theme)
         current-theme (deref current-theme*)
         lib (make-lib-with-theme current-theme ordered-token-sets)
 
+        has-back-button? (contains? #{:empty-themes :themes-overview} (:prev-type state))
```

Comment:
I'd prefer not leaking state here. My proposal is to add a parameter like `is-first-view` that is calculated by the caller, looking if `:prev-type` is empty. And here, show the back button only if it's not the first view.